tools/pbstate/src/main.c

```diff
@@ -134,10 +134,15 @@ int main(int argc, char * const argv[])
         }
     }
 
+#if !defined(PRIMARY_PART) && !defined(BACKUP_PART)
```

Comment:
I think this should be !defined || !defined (that is, only use the default path below if both are defined). Possibly expressed as !(defined && defined).

Reply:
fixed